torchvision/ops/focal_loss.py

```diff
@@ -16,19 +16,20 @@ def sigmoid_focal_loss(
     Loss used in RetinaNet for dense detection: https://arxiv.org/abs/1708.02002.
 
     Args:
-        inputs: A float tensor of arbitrary shape.
+        inputs (Tensor) : A float tensor of arbitrary shape.
```

Comment:
Sorry for nitpicking but we should keep the styles consistent across the code:
```suggestion
        inputs (Tensor): A float tensor of arbitrary shape.
```

Here is an example of the styles we use for reference:
https://github.com/pytorch/vision/blob/289fce29b3e2392114aadbe7a419df0f2e3ac1be/torchvision/utils.py#L32-L55

Reply:
You are right. I don't think we have been very consistent with the use of `optional` in docstrings. A common use in the codebase is to indicate that `None` is a valid value. Less commonly it's also used when the parameter can omitted because it has a default value (keyword argument). @NicolasHug do you have any thoughts on this?